lib/matplotlib/figure.py

```diff
@@ -964,13 +965,44 @@ def set_facecolor(self, color):
         """
         self.patch.set_facecolor(color)
 
-    def set_dpi(self, val):
+    def set_dpi(self, dpi):
```

Comment:
Please don't change argument names, it will break people doing `**kwargs` into it.